Cargo.toml

```diff
@@ -42,6 +42,7 @@ with_ctap1 = ["crypto/with_ctap1", "opensk/with_ctap1"]
 with_nfc = ["libtock_drivers/with_nfc"]
 vendor_hid = ["opensk/vendor_hid"]
 ed25519 = ["ed25519-compact", "opensk/ed25519"]
+rust_crypto = ["opensk/rust_crypto"]
```

Comment:
At the end of this file, you can add the following for crates you want to optimize. This will trade with binary size, so only select those that matter. Might not work though, so needs testing.

```toml
[profile.release.package]
aes = { opt-level = 3 }
p256 = { opt-level = 3 }
sha2 = { opt-level = 3 }
```

Reply:
First time I try this, so I'm learning. Apparently this is not recursive, so you have to specify the relevant dependencies too. I think we need to add also those crates to the list of `opt-level = 3`:
- `ecdsa`
- `elliptic-curve`
- `crypto-bigint`
- maybe `primeorder`
- maybe `rand_core`
- maybe `subtle`

Or maybe a subset of those is enough, or maybe I'm still missing some that are important.

I don't think it's worth trying to optimize too much though, so I would not try more than once.

Reply:
1% faster for `Ecdsa::SecretKey::sign`, no effect on anything else, but we lose more binary size. So I suggest we keep it as is!

Reply:
Sounds good